### DISCUSSION TITLE
default Microsoft Edge WebView2

### DISCUSSION THREAD

**Discussion by Lardon44** [MicrosoftEdge/WebView2Feedback#4992] [Q&A]
Hello,
I installed "MicrosoftEdgeWebView2RuntimeInstallerX64.exe"
Then I launched "WebView2APISample.exe"
The window opens and I have the message "failed to create webview: 0x8008005"
Thanks for your help
Best regards

**Comment by sivMSFT** [comment]
Hi @Lardon44,

Thank you for reporting the issue.

Could you please provide more details about the problem you're encountering while filtering requests with a custom host? Specifically:

- The steps you have already taken.
- Any error messages or logs you have received.
- The expected behavior versus the actual behavior.

Additionally, could you let us know if the problem still persists after any recent changes or troubleshooting steps you've tried?
